Bio/Seq.py

```diff
@@ -3049,10 +3252,15 @@ def __bytes__(self):
             return b""
         raise UndefinedSequenceError("Sequence content is undefined")
 
+    def __add__(self, other):
+        if isinstance(other, _UndefinedSequenceData):
+            return _UndefinedSequenceData(self._length + other._length)
+        raise TypeError
+
```

Comment:
This breaking of the add method means I can think of a clearly different use case for the now deprecated ``UnknownSeq``, things like ``UnknownSeq(100, character="N")`` versus ``Seq("N"*100)`` for gaps when joining contigs together. The workaround here is obvious though - make the short spacer sequence explicit.